libr/core/cmd_open.c

```diff
@@ -1373,20 +1400,49 @@ static int cmd_open(void *data, const char *input) {
 	case 'p': // "op"
 		/* handle prioritize */
 		if (input[1]) {
-			int fd = r_num_math (core->num, input + 1);
-			if (fd >= 0 || input[1] == '0') {
-				RIODesc *desc = r_io_desc_get (core->io, fd);
-				if (desc) {
-					// only useful for io.va=0
-					// load bininfo for given fd
-					r_core_cmdf (core, "obo %d", fd);
-					core->io->desc = desc; // XXX we should use fd here, not *pointer
-					r_core_block_read (core);
-				} else {
-					eprintf ("Cannot find RBinFile associated with fd %d\n", fd);
+			switch (input[1]) {
+			case '?': // "op?"
+				eprintf ("Usage: op[rnp] [fd]\n");
+				eprintf (" opr   = open next file rotating if not found\n");
+				eprintf (" opn   = open next file\n");
+				eprintf (" opp   = open previous file\n");
+				eprintf (" op 3  = opep priorizing fd = 3\n");
```

Comment:
typo `opep`

Reply:
Is this command really necessary? I think there is already another one that does mostly the same thing, no?

Reply:
which commands do the same?